cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/preview/DefaultPreviewRunner.java

```diff
@@ -177,6 +178,8 @@ public Future<PreviewRequest> startPreview(PreviewRequest previewRequest) throws
     DataTracerFactoryProvider.setDataTracerFactory(preview, dataTracerFactory);
 
     String config = request.getConfig() == null ? null : GSON.toJson(request.getConfig());
+    LinkedTreeMap<String, String> versionRequestObject = (LinkedTreeMap<String, String>) request.getVersion();
```

Comment:
can you point me to the design of this version object?

Reply:
Currently only change summary is present in the version request object. This can change in the future
```
"version": {
      "changeSummary": "text to describe the change of the version"
  }
```

Reply:
same here, this shouldn't called version and it should be an object instead of map